src/duckling/canvas/drawing/drawn-construct.ts

```diff
@@ -125,6 +127,35 @@ export function drawnConstructBounds(drawnConstruct : DrawnConstruct) : Box2 {
     return bounds;
 }
 
+/**
+ * Set the position of a drawn construct
+ * @param  drawable Drawn construct to set the position of
+ * @param  position Position to apply to the drawn construct
+ */
+export function setConstructPosition(drawable : DrawnConstruct, position : Vector) {
+    function _setDisplayObjectPosition(displayObject : DisplayObject) {
```

Comment:
Take this function out. V8 doesn't optimize t his behavior yet so it leads to an allocation every time.